packages/toolpad-app/src/toolpadDataSources/rest/server.ts

```diff
@@ -137,20 +137,22 @@ async function execBase(
   connection: Maybe<RestConnectionParams>,
   fetchQuery: FetchQuery,
   params: Record<string, string>,
+  envParams?: Record<string, string>,
```

Comment:
Maybe we don't need this `envParams` argument after all? 
I think `resolveBindableEntries` already resolves env variable parameters by itself. Then we can just pass an empty object `{}` as the scope for the `resolveBindableEntries` below.